src/__tests__/index.ts

```diff
@@ -213,6 +198,21 @@ const tests: Record<string, TestCase> = {
       {favoriteIceCream: ['mint', 'chocolate']},
     ],
   },
+  'can handle nested keys that are an array of values with a wildcard': {
+    input: [
+      [
+        {favorite: {iceCream: ['mint', 'chocolate']}},
+        {favorite: {iceCream: ['candy cane', 'brownie']}},
+        {favorite: {iceCream: ['birthday cake', 'rocky road', 'strawberry']}},
+      ],
+      'cc',
+      {keys: ['favorite.iceCream.*']},
```

Comment:
I'm fine with this working, but I'd also like to support `favorite.iceCream` here as well. We should be able to process an `item` of `String | Array<String>` and `favorite.icCream` is `Array<String>`.